tenets/codelingo/code-review-comments/avoid-renaming-imports/expected.json

```diff
@@ -0,0 +1,32 @@
+[
```

Comment:
Getting duplicate hits. Please link to this in your tenet card.

https://github.com/mullikine/mullikine.github.io/blob/master/codelingo/troubleshooting/tenets/avoid-renaming-imports_multiple-review-hits.org